examples/MagicWeatherCompose/build.gradle

```diff
@@ -0,0 +1,6 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+    id 'com.android.application' version '8.0.2' apply false
+    id 'com.android.library' version '8.0.2' apply false
+    id 'org.jetbrains.kotlin.android' version '1.7.20' apply false
+}
```

Comment:
we can use the gradle library catalogs, but we can do it in another PR

Reply:
Yeah I thought about that. Going to make that change in another PR, so we are using catalogs everywhere

Reply:
Done in #1081